### ISSUE TITLE
How to do regression using gradient?

### ISSUE THREAD

**Issue by cubicgate** [tensorflow/quantum#488]
Is it possible to add one example of using gradient to do regression (for example, on sine curves) at the end of these gradient examples in https://www.tensorflow.org/quantum/tutorials/gradients? I would like to learn how to use the gradient after getting the gradient.

   Thanks!

**Comment by MichaelBroughton** [comment]
Good Idea, perhaps we could put another small section at the end of the tutorial showcasing a quick use case of the differentiator you just made. Let's leave this open in case anyone would like to take this on.

**Comment by lockwo** [comment]
A tutorial hasn't been made, but there are examples of regression (e.g. https://github.com/lockwo/quantum_computation/blob/master/TFQ/vqc/boston_housing.py). Are you still looking for more information?
